### ISSUE TITLE
[0.37.0] failed test

### ISSUE THREAD

**Issue by remicollet** [DataDog/dd-trace-php#717]
```
TEST 140/156 [tests/ext/sandbox/memory_limit_graceful_bailout.phpt]
========DIFF========
003+ Fatal error: Allowed memory size of 2097152 bytes exhausted (tried to allocate 80 bytes) in /dev/shm/BUILD/php56-php-pecl-datadog-trace-0.37.0/NTS/tests/ext/sandbox/memory_limit_graceful_bailout.php on line 3
003- PHP Fatal error:  Allowed memory size of 2097152 bytes exhausted %s (tried to allocate %d bytes) in %s on line %d
========DONE========
FAIL The tracer bails out gracefully when memory_limit INI is reached in shutdown hook [tests/ext/sandbox/memory_limit_graceful_bailout.phpt] 

```

**Comment by remicollet** [comment]
nothing critical... only about error formating

**Comment by SammyK** [comment]
Thanks for the report @remicollet! This was fixed as part of #723 and released in v0.38.0.
